react/ProgressionBanner/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withStyles } from '@material-ui/core/styles'
+
+import Banner from '../Banner'
+import { LinearProgress } from '../Progress'
+import { paleGrey } from '../palette'
+
+const styles = {
+  root: {
+    backgroundColor: 'transparent',
+    height: '0.125rem',
+    marginTop: '-0.125rem'
+  }
+}
+
+const ProgressionBanner = withStyles(styles)(
+  ({ classes, value, text, icon, button, noProgressBar }) => {
```

Comment:
Instead of noProgressBar I'd prefer to have progressBar: false with a default prop progressBar of true.